email/1.3/modules/ROOT/edits/email-studio-configuration.adoc

```diff
@@ -0,0 +1,147 @@
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add an operation for Email Connector, follow these steps:
+
+. In the *Mule Palette* view, select *Email* and then select the desired operation.
+. Drag the operation onto the Studio canvas and to the right of the input source.
+
+The following screenshot shows the Email Connector operations in the Mule Palette of Anypoint Studio:
+
+.Email Connector Operations
+image::email-connector-operations.png[Email Connector Operations]
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. Configuring a global element for Email Connector requires you to set a configuration based on the supported protocols for the connector operations:
+
+[%header,cols="20s,80a"]
+|===
+| Global Element Configuration | Connector Operation/Sources
+| IMAP (Internet Message Access Protocol) a| * *Delete*
+* *Expunge Folder*
+* *List - IMAP*
+* *Mark As Deleted*
+* *Mark As Read*
+* *On New Email - IMAP*
+| POP3 (Post Office Protocol 3) a| * *List POP3*
+* *On New Email - POP3*
+| SMTP (Simple Mail Transfer Protocol) a| * *Send*
+|===
+
+For example, to configure a POP3 global element for the *List POP3* operation, follow these steps:
+
+. Select the name of the connector in the Studio canvas.
+. Select the *List POP3* operation in the Studio canvas.
+. In the *List POP3* configuration screen for the operation, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
+On the *Email POP3* window, for *Connection* select any of the connection types to provide to this configuration:
+
+* *POP3 Connection*
+* *POP3S Connection*
+
+[start=6]
+. On the *General* tab, specify the connection information for the connector, such as *Host*, *Port* and *TLS configuration*.
+. On the *Advanced* tab, optionally specify timeout configuration and reconnection information, including a reconnection strategy.
+. Click *OK*.
+
+The following screenshot shows the Email Connector Global Element Configuration window in Anypoint Studio:
+
+.Email Connector Global Element Configuration
+image::email-global-configuration.png[Email Connector Global Element Configuration]
+
+[[configure-other-fields]]
+== Configure Additional Connector Fields
+
+For IMAPS, POP3S, and SMPTPs protocol connections, you can use Transport Layer Security (TLS) and configure email by providing a key store with your certificate. Additionally, you can also enable two-way authentication by providing a trust store. For details, see xref:email-documentation.adoc#Tls[Email Connector Reference] and xref:mule-runtime::tls-configuration.adoc[Configure TLS with Keystores and Truststores].
```

Comment:
"Additionally, you can also enable..." is redundant. Omit one or the other of "additionally" or "also."
If this is in the template, please fix the template.

Reply:
```suggestion
For IMAPS, POP3S, and SMPTPs protocol connections, you can use Transport Layer Security (TLS) and configure email by providing a keystore with your certificate. Additionally, you can enable two-way authentication by providing a truststore. For details, see xref:email-documentation.adoc#Tls[Email Connector Reference] and xref:mule-runtime::tls-configuration.adoc[Configure TLS with Keystores and Truststores].
```

Reply:
Ditto re. template; note that the text was out of sync with the presentations of keystore and truststore in the xref.